Google.Api.Generator/CodeGenerator.cs

```diff
@@ -138,7 +138,7 @@ internal static class CodeGenerator
                     // Generate snippets for the service
                     // TODO: Consider removing this once we have integrated the standalone snippets
                     // with docs generation.
-                    var snippetCtx = SourceFileContext.CreateUnaliased(clock, s_wellknownNamespaceAliases, s_avoidAliasingNamespaceRegex);
+                    var snippetCtx = SourceFileContext.CreateUnaliased(clock, s_wellknownNamespaceAliases, s_avoidAliasingNamespaceRegex, true);
```

Comment:
It might be worth using named arguments both here and for the other call, just for clarity.

Reply:
Yes, done.